homeassistant/components/fivem/sensor.py

```diff
@@ -0,0 +1,151 @@
+        device_class: str = None,
+    ) -> None:
+        """Initialize sensor base entity."""
+        super().__init__(server, type_name, icon, device_class)
+        self._state: Any = None
+        self._unit = unit
+
+    @property
+    def available(self) -> bool:
+        """Return sensor availability."""
+        return self._fivem.online
+
+    @property
+    def native_value(self):
+        """Return sensor state."""
+        return self._state
+
+    @property
+    def native_unit_of_measurement(self) -> str:
+        """Return sensor measurement unit."""
+        return self._unit
+
+
+class FiveMPlayersOnlineSensor(FiveMSensorEntity):
+    """Representation of a FiveM online players sensor."""
+
+    def __init__(
+        self,
+        server: FiveMServer,
+    ) -> None:
+        """Initialize online players sensor."""
+        super().__init__(
+            server, NAME_PLAYERS_ONLINE, ICON_PLAYERS_ONLINE, UNIT_PLAYERS_ONLINE
+        )
+        self._extra_state_attributes: dict[str, Any] = {}
+
+    async def async_update(self) -> None:
+        """Update online players state and device attributes."""
+        self._state = self._fivem.players_online
+
+        extra_state_attributes = {}
+        players_list = self._fivem.players_list
+
+        if len(players_list) != 0:
+            extra_state_attributes = {ATTR_PLAYERS_LIST: players_list}
+
+        self._extra_state_attributes = extra_state_attributes
+
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Return players list in device attributes."""
+        return self._extra_state_attributes
+
+
+class FiveMPlayersMaxSensor(FiveMSensorEntity):
+    """Representation of a FiveM maximum number of players sensor."""
+
+    def __init__(self, server: FiveMServer) -> None:
+        """Initialize maximum number of players sensor."""
+        super().__init__(
+            server=server,
+            type_name=NAME_PLAYERS_MAX,
+            icon=ICON_PLAYERS_MAX,
+            unit=UNIT_PLAYERS_MAX,
+        )
+
+    async def async_update(self) -> None:
+        """Update maximum number of players."""
+        self._state = self._fivem.players_max
+
+
+class FiveMResourcesSensor(FiveMSensorEntity):
+    """Representation of a FiveM resources sensor."""
+
+    def __init__(
+        self,
+        server: FiveMServer,
+    ) -> None:
+        """Initialize resources sensor."""
+        super().__init__(server, NAME_RESOURCES, ICON_RESOURCES, UNIT_RESOURCES)
+        self._extra_state_attributes: dict[str, Any] = {}
+
+    async def async_update(self) -> None:
+        """Update resources state and state attributes."""
+        self._state = self._fivem.resources_count
+
+        extra_state_attributes = {}
+        resources_list = self._fivem.resources_list
+
+        if len(resources_list) != 0:
+            extra_state_attributes = {ATTR_RESOURCES_LIST: resources_list}
+
+        self._extra_state_attributes = extra_state_attributes
```

Comment:
```suggestion
        self._attr_extra_state_attributes = extra_state_attributes
```
By using short hand attributes you can remove the `extra_state_attributes` property